tests/Traits/TerminusTestTrait.php

```diff
@@ -155,7 +155,13 @@ protected function getPluginsDir(): string
      */
     protected function getDependenciesBaseDir(): string
     {
-        return getenv('TERMINUS_DEPENDENCIES_BASE_DIR');
+        global $terminusPluginsDependenciesVersion;
```

Comment:
It's an implementation detail that this version number is stored as a global; we don't want to break the functional tests if we need or want to change this implementation detail in the future. Instead, maybe we could add an `--all` or wildcard feature to self:plugin:remove, and have the tests rely on that. The tests could also create a fresh terminus config directory on every run, in which case you would also be able to guarantee that the dependencies base dir is empty when you start.

Reply:
@greg-1-anderson fixed